### ISSUE TITLE
A suggestion is to increase the maximum number of threads to 256

### ISSUE THREAD

**Issue by scyou2** [fitztrev/lichess-tauri#46]
The maximum number of available threads for the 'Liches Local Engine' is 64 It's a bit rare, as many people use over 200 CPU threads. 
A suggestion is to increase the maximum number of threads to 256 

Reference information: 
threads licess.org Maximum number of available threads. is 65536 
https://lichess.org/api#tag/External -Engine

**Comment by scyou2** [comment]
After switching to a different computer, I found that the maximum number of threads varies.
I have found the problem. My computer has 256 threads, but 'Liches Local Engine' only recognizes 64 threads, so the software can only set a maximum of 64 threads.
The problem here is that the CPU has 4 Numa Groups, each with 64 threads, and 'Liches Local Engine' should only recognize one Numa Group.

I suggest you update in the next version to use all CPU threads.

**Comment by Disservin** [comment]
> 64 It's a bit rare

Not really, windows only really supports 64 threads, if you go higher than that the software you are using needs to be aware of that and change how it is working, which is usually more effort. 
However I don't see why this is the case here, apparently something sets the affinity which shouldnt be the case.

https://learn.microsoft.com/en-us/windows/win32/procthread/processor-groups

If you want to try to give Stockfish 17 a try and set the `NumaPolicy` option to `hardware` and check if this solves the issue temporarily.

**Comment by scyou2** [comment]
Thank you for your reply. You are really professional.
Can you provide a suggestion
The 'Liches Local Engine' option in the dropdown menu of threads has been changed to allow for direct input of numbers.
You can simultaneously scroll down the number of threads and directly fill in the number of threads.

**Comment by scyou2** [comment]
And now I use the stockfish 15 dev.  I think it's an important information.

**Comment by Disservin** [comment]
Ah it might be that you can't yet set other options in lichess-tauri :/

**Comment by minskcity** [comment]
Was this feature ever added? I have 192 cores and would love to use more than 64 threads...
